ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -65,17 +65,10 @@ public async Task<IActionResult> OnPostSubmitAsync()
 
         public async Task<IActionResult> OnPostCreateLinkAsync()
         {
-            // First Validate and Save current page details
```

Comment:
I first thought this would be useful (taken from the OnPostSubmit) behaviour but it caused an exception when called from the overview page (where validateAndSave throws). But that made me think that it doesn't really make sense to autosave when you navigate away from the page to create a new notification - or at least it's not consistent with the rest of the page, we never save when we navigate away. For submission it obviously does make sense.

Reply:
That seems like the right call to me, in fact I think that actually raises a further questions - do we want this button on non-overview pages in the first place?